### PR TITLE
Added 2 new questions (Q69-Q70).

### PR DESCRIPTION
**Pull request by dianosaur12**

### CODE REVIEW

File: it-operations/it-operations-quiz.md [hunk 1]
```diff
@@ -568,3 +568,17 @@ To get to either UEFI or BIOS on a system you usually go via the computer's boot
 - [ ] It has been integrated into the southbridge to provide support for the CPU.
 - [ ] It is the peripheral and I/O controller, connected to the processor via the southbridge.
 - [ ] It provides mathematical and 3D physics support for the CPU in the absence of a GPU.
+
+#### Q69. Which characters are not allowed in a filename?
```
Comment (artemu78) [inline it-operations/it-operations-quiz.md:572]: Maybe it is worth mentioning the exact OS you speak about.

File: it-operations/it-operations-quiz.md [hunk 2]
```diff
@@ -568,3 +568,17 @@ To get to either UEFI or BIOS on a system you usually go via the computer's boot
...
+
+- [ ] anything other than a letter, number, or space 
+- [ ] spaces, periods, and commas
+- [x] \, /, :, ?, and *
+- [ ] !, ^, &, (,), and "
+
+#### Q70. What is the most direct way to change the desktop background?
```
Comment (artemu78) [inline it-operations/it-operations-quiz.md:579]:
```suggestion
#### Q70. What is the most direct way to change the desktop background in Windows OS?
```

File: it-operations/it-operations-quiz.md (hunk 1 above)
Comment (dianosaur12) [reply it-operations/it-operations-quiz.md:572]: > Maybe it is worth mentioning the exact OS you speak about.

While I would usually agree with that, I think the options provided below the question already do a decent job of giving the user an idea of which OS is being referenced here, which is why I didn't see a need to further specify the OS. Thank you for pointing it out, though!

Comment (artemu78) [reply it-operations/it-operations-quiz.md:572]: Yeah, maybe you right

**Comment by dianosaur12** [comment]
Hi,  @Ebazhanov! Added whatever necessary suggestions/edits that were required to the file, could you please merge it with main now? Thanks!

**Review by Ebazhanov — APPROVED** [review]
LGTM, thanks a lot @dianosaur12 👍

**Pull request merged**